src/store/reducers/packagesReducer.js

```diff
@@ -106,6 +107,7 @@ export const redux = createReducerScenario({
   deletePackage: { type: 'PACKAGES_DELETE', reducer: deletePackageReducer },
   updatePackages: { type: 'PACKAGES_UPDATE', reducer: updatePackagesReducer },
   registerError: { type: 'PACKAGES_REDUCER_ERROR', reducer: errorReducer },
+  resetPendingAndError: { type: 'PACKAGES_REDUCER_RESET_ERROR_PENDING', reducer: resetPendingAndErrorReducer },
```

Comment:
Why is "REDUCER" part of the name here? Its not in any of the others.

Reply:
Do you mean the value for type?